lib/sensors/mux.hpp

```diff
@@ -13,6 +13,7 @@ static constexpr std::uint8_t kDefaultMuxAddress = 0x70;
 static constexpr std::uint8_t kMaxNumChannels    = 8;
 static constexpr core::Float kFailureThreshold
   = 0.25;  // TODOLater: finalize this value with Electronics
+std::uint8_t num_unusable_sensors = 0;
```

Comment:
no, no, no, no! don't put it here. now it's a program-wide global variable. put it inside the function!

Reply:
oh whoops, will fix asap.